rllib/algorithms/tests/test_callbacks.py

```diff
@@ -102,39 +103,48 @@ def on_episode_created(
 class TestCallbacks(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
-        ray.init()
+        ray.init(num_cpus=12)
```

Comment:
Please don’t add num cpus. On workspaces you cannot run this code.

Reply:
done